flowzone.yml

```diff
@@ -53,6 +53,41 @@
       submodules: "recursive"
       token: ${{ secrets.FLOWZONE_TOKEN }}
 
+  - &ifInternalPullRequest
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+  - &ifPrivateRepository
+    if: github.event.repository.private
+
+  - &loginWithDockerHub
+    name: Login to Docker Hub
+    continue-on-error: true
+    uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9 # v1
```

Comment:
This updates from `f4ef78c080cd8ba55a85445d5b36e214a81df20a` for a bunch of the uses - is that fine/safe?

Reply:
Good catch, they should be at v2, this is actually an accidental downgrade.